src/qibojit/backends/clifford_operations_cpu.py

```diff
@@ -3,12 +3,22 @@
 import numpy as np
 from numba import njit, prange, uint64
 
+PARALLEL = False
```

Comment:
Are you sure we actually need this?

Reply:
No I am not actually, I was keeping it just in case I found a way to get back the speed up with parallelization enabled.